upstart/upstart_install.sh

```diff
@@ -5,12 +5,15 @@ if [ "$HOSTNAME" != "$ROBOT-b1" ]; then
 	exit
 fi
 
-
+sudo apt-get install ros-indigo-robot-upstart
```

Comment:
we should make this a dependency of ros-indigo-care-o-bot-robot. can you please open a PR to github.com/ipa320/care-o-bot?